aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java

```diff
@@ -367,13 +367,13 @@ private byte[] compileClass(String className, FunctionSection functions) {
         var classWriter = new ClassWriter(ClassWriter.COMPUTE_FRAMES);
         classWriter.visit(
                 Opcodes.V11,
-                Opcodes.ACC_PUBLIC | Opcodes.ACC_FINAL,
+                Opcodes.ACC_PUBLIC | Opcodes.ACC_FINAL | Opcodes.ACC_SUPER,
```

Comment:
Apparently, compatibility with project Leyden.

Reply:
Not Leyden, Valhalla. In Valhalla, if you have a class without ACC_SUPER (renamed ACC_IDENTITY), you are creating a value class. See https://bugs.openjdk.org/browse/JDK-8317278

Reply:
Many thanks for the explanation!

Reply:
I'm guessing this doesn't matter was we're currently generating `V11` classes, but good to fix for the future!